doc/howto/images_copy.md

```diff
@@ -0,0 +1,88 @@
+(images-copy)=
+# How to copy and import images
+
+To add images to an image store, you can either copy them from another server or import them from files (either local files or files on a web server).
+
+## Copy an image from a remote
+
+To copy an image from one server to another, enter the following command:
+
+    lxc image copy [<source_remote>:]<image> <target_remote>:
+
+```{note}
+To copy the image to your local image store, specify `local:` as the target remote.
+```
+
+See `lxc image copy --help` for a list of all available flags.
+The most relevant ones are:
+
+`--alias`
+: Assign an alias to the copy of the image.
+
+`--copy-aliases`
+: Copy the aliases that the source image has.
+
+`--auto-update`
+: Keep the copy up-to-date with the original image.
+
+`--vm`
+: Copy an image that can be used to create virtual machines.
```

Comment:
Is that correct?
When launching an instance, I don't need to select a different image depending on whether I want to start a container or a VM. But my guess is that LXD requests a different version of the image under the hood, and that's why I need to be more specific when copying?

Reply:
Yes it is correct. When launching an instance you do need to provide the `--vm` flag, which under the hood also passes the equivalent indicator to the image selection process (instance type). So here as we are operating only on images, we do need a `--vm` flag in order to differentiate container images from VM images when referring to them by alias.